packages/web/src/utils/trpcClientWeb.ts

```diff
@@ -24,9 +26,7 @@ export function createAudiusTrpcClient(currentUserId: number | null) {
 // since tRPC server is deployed manually atm.
 // in the future some tRPC middleware can set host to currently selected DN per request
 function getTrpcEndpoint() {
-  if (process.env.VITE_TRPC_ENDPOINT) return process.env.VITE_TRPC_ENDPOINT
```

Comment:
nit: We should change the below logic to use a `env.TRPC_ENDPOINT` instead imo, even if it's temporary

Reply:
nice